core/src/main/scala/org/apache/spark/rdd/ReliableCheckpointRDD.scala

```diff
@@ -100,10 +107,52 @@ private[spark] object ReliableCheckpointRDD extends Logging {
     "part-%05d".format(partitionIndex)
   }
 
+  private def checkpointPartitionerFileName(): String = {
+    "_partitioner"
+  }
+
   /**
-   * Write this partition's values to a checkpoint file.
+   * Write RDD to checkpoint files and return a ReliableCheckpointRDD representing the RDD.
    */
-  def writeCheckpointFile[T: ClassTag](
+  def createCheckpointedRDD[T: ClassTag](
```

Comment:
you can probably just call this `create`, since you call it like `ReliableCheckpointRDD.create` so it's pretty clear what you're making